src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11001,6 +11113,64 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ActionBodyExecTemplate'
+  RuleSetPaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleSetEntry'
+  RuleSetEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/RuleSet'
+  RuleSet:
+    type: object
+    required:
+      - id
+      - ownedBy
+    properties:
+      id:
+        description: Identifier for the rule set
+        type: string
+      ownedBy:
+        description: The identifier of the folder which owns the rule set
+        type: string
+      sharedWith:
+        description: List of folder identifiers which also use the rule set
+        type: array
+        items:
+          type: string
+  LinkedRuleSetBodyCreate:
+    type: object
+    required:
+      - id
+      - type
+    properties:
+      id:
+        description: Identifier for either the folder or the rule set
+        type: string
+      type:
```

Comment:
I don't think this is actually necessary.  The client and server both have access to enough information to determine what the type of the node is, and this adds in the potential for edge cases around mismatches parameters.